src/main/java/use_cases/_common/authentication/AuthenticationInterface.java

```diff
@@ -0,0 +1,18 @@
+package use_cases._common.authentication;
+
+import entity.User;
+
+/**
+ * Interface for authentication service.
+ * Defines methods for user authentication and session management.
+ */
+public interface AuthenticationInterface {
+
+    boolean authenticate(String userEmail, String userPassword);
+
+    User getLoggedInUser();
+
+    void logout(User user);
+
+    void setLoggedInUser(User user); // Ensure this method is in the interface
```

Comment:
the comment here seems either not needed or misworded - won't the interface ensure the method is there?

Reply:
i can take out that comment